packages/base/components/Authored/AuthoredHead.vue

```diff
@@ -0,0 +1,78 @@
+<script setup>
+import parseMarkdown from "@/utils/markdown/parse.js";
+
+const props = defineProps({
+  title: {
+    type: String,
+    required: true,
+  },
+
+  description: {
+    type: String,
+    default: "",
+  },
+
+  exhibitionTitle: {
+    type: String,
+    default: "",
+  },
+
+  hero: {
+    type: Object,
+    default: null,
+  },
+
+  contextLabel: {
+    type: String,
+    default: null,
+  },
+});
+
+const heroImage = computed(() => (props.hero ? props.hero.image : null));
+const heroImageAlt = computed(() => heroImage.value?.description || "");
+</script>
+<template>
+  <div class="container mt-3">
+    <div class="row justify-content-center">
+      <div class="col col-12 col-lg-8 mb-md-3 mb-lg-4">
+        <article>
```

Comment:
[article](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/article) is not appropriate here